atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/sweep/SweeperServiceImpl.java

```diff
@@ -126,16 +126,18 @@ private SweepBatchConfig buildConfigWithOverrides(
     }
 
     private TableReference getTableRef(String tableName) {
-        WebPreconditions.checkArgument(
-                TableReference.isFullyQualifiedName(tableName), "Table name {} is not fully qualified", tableName);
+        Preconditions.checkArgument(
+                TableReference.isFullyQualifiedName(tableName),
+                "Table name is not fully qualified",
+                UnsafeArg.of("tableName", tableName));
         return TableReference.createFromFullyQualifiedName(tableName);
```

Comment:
nit: we should clean up `TableReference` to use the `com.palantir.logsafe.Preconditions` instead of Guava, then we could get rid of this `getTableRef(String)` method and just inline the `TableReference.createFromFullyQualifiedName(tableName)`

```
    public static TableReference createFromFullyQualifiedName(String fullTableName) {
        int index = fullTableName.indexOf('.');
        Preconditions.checkArgument(
                index > 0, "Table name is not a fully qualified table name.", UnsafeArg.of("tableName", fullTableName));
        return create(
                Namespace.create(fullTableName.substring(0, index), Namespace.UNCHECKED_NAME),
                fullTableName.substring(index + 1));
    }
```

Reply:
Do we have errorprones which will automatically perform that migration?

Reply:
Negative Ghostrider. I ran [`./gradlew compileJava compileTestJava -PerrorProneApply=PreferSafeLoggingPreconditions`](https://github.com/palantir/gradle-baseline/blob/develop/baseline-error-prone/src/main/java/com/palantir/baseline/errorprone/PreferSafeLoggingPreconditions.java) against `develop` but that didn't generate any changes as I imagine since the aren't `Arg`s we're hitting the path without suggested fix. Separately we could make that smarter and generate `UnsafeArg`s, but don't think we should block on this.

https://github.com/palantir/gradle-baseline/blob/828111050da72901a66a676ae517719a9e9d9849/baseline-error-prone/src/main/java/com/palantir/baseline/errorprone/PreferSafeLoggingPreconditions.java#L88-L120

Reply:
Draft PR https://github.com/palantir/atlasdb/pull/6223 to migrate more Guava -> logsafe Preconditions